Source/WebInspectorUI/UserInterface/Models/CSSGrouping.js

```diff
@@ -80,6 +105,31 @@ WI.CSSGrouping = class CSSGrouping
         console.assert(this.isMedia);
         return "@media";
     }
+
+    // Private
+
+    _handleChangeGroupingTextResult(groupingPayload)
+    {
+        console.assert(groupingPayload.type == this._type);
+
+        console.assert(groupingPayload.ruleId);
+        console.assert(groupingPayload.text);
+
+        this._id = groupingPayload.ruleId;
+        this._text = groupingPayload.text;
+
+        let location = {};
+        if (groupingPayload.sourceRange) {
+            location.line = groupingPayload.sourceRange.startLine;
+            location.column = groupingPayload.sourceRange.startColumn;
+            location.documentNode = this._nodeStyles.node.ownerDocument;
+        }
+
+        let groupingSourceCodeLocation = WI.DOMNodeStyles.createSourceCodeLocation(groupingPayload.sourceURL, location);
+        groupingSourceCodeLocation = WI.cssManager.styleSheetForIdentifier(this._id.styleSheetId).offsetSourceCodeLocation(groupingSourceCodeLocation);
```

Comment:
NIT: id prefer if we didn't reassign the same variable, as that can make it harder to debug things.  i'd inline this in the `this._sourceCodeLocation = ...` instead